### ISSUE TITLE
Broken after codecov.io updated

### ISSUE THREAD

**Issue by larshp** [larshp/abapCov#8]
Seems like the interface is not working anymore after codecov.io has been updated/upgraded. Perhaps they changed the API, need to investigate. Or perhaps I'm just using the wrong token.

Also the codecov.io page does not show the ABAP files anymore, looks like they have restricted the list of languages.

**Comment by pacroy** [comment]
When I ran ZABAPCOV, it went shortdump telling an assertion was met. Is this the same issue?

**Comment by larshp** [comment]
no, dont think so, what SAP_BASIS version are you running? note that I have only tested it on "Version requirement: 740 SP8" as far as I remember it did not work on SP6

**Comment by pacroy** [comment]
Thanks for clarification. It is version 750 SP1 on my system.
